cmd/prom-migrator/main.go

```diff
@@ -130,6 +131,8 @@ func parseFlags(conf *config, args []string) {
 		"Setting this value less than zero will indicate all data from mint upto now. ")
 	flag.StringVar(&conf.maxBlockSize, "max-read-size", "500MB", "(units: B, KB, MB, GB, TB, PB) the maximum size of data that should be read at a single time. "+
 		"More the read size, faster will be the migration but higher will be the memory usage. Example: 250MB.")
+	flag.IntVar(&conf.numShards, "num-shards", 4, "Number of shards involved in pushing a block. More the number of shards, more will be the number of"+
+		"parallel flushers involved per in-memory block and hence greater will be the through-put for any in-memory block.")
```

Comment:
```suggestion
	flag.IntVar(&conf.numShards, "num-shards", 4, "Number of shards to split a fetched block in. Each shard is written concurrently")
```

Reduce implementation details from the flag desc

Reply:
Definitely. However, along with this suggestion, I am adding another thing as, `Note: Larger shards count will lead to significant memory usages.`. I hope it is fine with the sentence.

Reply:
Yup, now that you mention it, Prometheus uses the same kind of warning on their remote_write config 👍